.github/workflows/dispatch-build-bottle.yml

```diff
@@ -203,15 +203,6 @@ jobs:
       BOTTLE_BRANCH: ${{github.actor}}/dispatch/${{inputs.formula}}/${{github.run_id}}
       BOTTLES_DIR: ${{ github.workspace }}/bottles
     steps:
-      - name: Check that we're not uploading Sequioa x86_64 bottles
-        run: |
-          if [[ "${DISPATCH_BUILD_BOTTLE_RUNNER}" =~ (^|,)15(-x86_64)?($|,) ]]
-          then
-            echo "::error ::Refusing to upload bottles for macOS 15 x86_64!"
-            echo "::error ::Please don't upload macOS 15 x86_64 bottles yet, @${DISPATCH_BUILD_BOTTLE_SENDER}."
-            exit 1
-          fi
```

Comment:
Do we want to add a check for the formulae we're ok with 15-x86_64 bottles being uploaded?

Reply:
Maybe though there's also no harm in it other than being a waste of time.

We will have logic for `determine-test-runners` though idk if that's really sharable here.

Reply:
Logic in `test-bot` will also likely need to be updated, as it will currently refuse to build a bottle if a formula has unbottled dependencies.

Reply:
Given it'll be just the GCC + LLVM dependency trees we can probably just bottle the whole tree.

Reply:
Happy to do that.